Keras_Tensorflow/04_DeployOnAKS.ipynb

```diff
@@ -198,38 +459,44 @@
       "      \"vnetSubnetId\": null\n",
       "    }\n",
       "  ],\n",
-      "  \"dnsPrefix\": \"fbAKSClust-fbaksrg-edf507\",\n",
-      "  \"fqdn\": \"fbaksclust-fbaksrg-edf507-97ac4d0f.hcp.eastus.azmk8s.io\",\n",
-      "  \"id\": \"/subscriptions/edf507a2-6235-46c5-b560-fd463ba2e771/resourcegroups/fbaksrg/providers/Microsoft.ContainerService/managedClusters/fbAKSCluster\",\n",
-      "  \"kubernetesVersion\": \"1.8.10\",\n",
+      "  \"dnsPrefix\": \"mabouaks-mabouaks-e984a9\",\n",
```

Comment:
Here is something we forgot which we need to do. We need to mask the subscriptionids like xxxxx-xxxx-xxxx-xxxx etc...I think the output is valuable that is why we don't want clean output here. I typically do that by editing the file after the merge directly on github so we can take care of that later if you cant find a way to do that. However, maybe opening notebook as text or something and then masking should also work although I didn't try that.

Reply:
Yes, the notebooks can be edited as text.